rlclientlib/logger/async_batcher.h

```diff
@@ -136,11 +144,12 @@ namespace reinforcement_learning { namespace logger {
   int async_batcher<TEvent, TSerializer>::fill_buffer(
                                                       std::shared_ptr<utility::data_buffer>& buffer,
                                                       size_t& remaining,
-                                                      api_status* status)
+                                                      api_status* status,
+                                                      unsigned int& original_count)
```

Comment:
this repo has convention of having "api_status* status" argument as the last one